content/integrations/docker.md

```diff
@@ -19,7 +19,7 @@ Get metrics from Docker in real time to:
 
 # Installation
 
-There are three ways to setup the Docker integration: install the agent on the host, on a single privileged container, and on each individual container.
+There are two ways to setup the Docker integration: install the agent on the host or on a single sidecar container.
```

Comment:
but there is that 3rd way to install. in each container. There was a specific scenario where support was recommending that (certain metrics not being collected). you lose out on the container pricing since you have so many agents installed, but its an option.